test/src/js/config.spec.js

```diff
@@ -1,11 +1,9 @@
 'use strict';
 
-const { expect } = require('chai');
-
 const underTest = require('../../../src/js/config');
 
 describe('./src/js/config', function () {
-	it('config should be an Object', function () {
-		expect(typeof underTest).to.equal('object');
+	test('config should be an Object', function () {
+		expect(typeof underTest).toEqual('object');
```

Comment:
All other tests are `toBe` variety. Is there a reason this is a `toEqual`?

Reply:
👍